app/boards/intel_adsp_ace20_lnl.conf

```diff
@@ -36,7 +36,7 @@ CONFIG_PCM_CONVERTER_FORMAT_FLOAT=y
 # SOF / audio modules / mocks
 # This mock is part of official sof-bin releases because the CI that
 # tests it can't use extra CONFIGs. See #9410, #8722 and #9386
-CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
+CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=m
```

Comment:
on LNL we so far only enable DTC as a loadable modules, so this *may* remove this module from the binary distribution